parser.y

```diff
@@ -1,13 +1,26 @@
 %include {
     #include <assert.h>
+    #include <stdio.h>
 }
 
-start ::= statement_list.
+%syntax_error {
+    fprintf(stderr, "%s", "Syntax Error!\n");
+}
+
+%start_symbol start
+
+start ::= header statement_list.
+start ::= error. {
+    fprintf(stderr, "%s", "Parser Error!\n");
+}
+
+header ::= PERCENT statement_list PERCENT. {
```

Comment:
This makes the header.

Reply:
Is this necessary for the prototype? Can this happen in the files that will be used first?

Reply:
The sample files, which are exported from kicad, use M48 for the header start and % for header end. My merger uses % and %. Its not necessary, i think testing would be easier if it implements all permutations here.